pkgs/tools/networking/dd-agent/trace-deps.nix

```diff
@@ -0,0 +1,73 @@
+[ { goPackagePath = "github.com/cihub/seelog";
```

Comment:
It was written by hand as I couldn't get go2nix to work on their setup as they use Rake or something and I'm not a Go guy to try and figure it out.